content/numpy/concepts/math-methods/terms/round/round.md

```diff
@@ -0,0 +1,155 @@
+---
+Title: '.round()' 
+Description: 'Rounds a number or an array of numbers to a specified number of decimal places.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'NumPy'
+  - 'Math'
+  - 'Arrays'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+  - 'paths/data-science-foundations'
+---
+
+**`.round()`** is a function from the NumPy Python library. It rounds a number or an array of numbers to a specified number of decimal places.
+
+## Syntax
+
+```psuedo
+numpy.round(array/number, decimal place, out)
+```
+
+- `array/number` is the number or array, whose elements (either a float or integer) will be rounded.
+- `decimal place` is the number of decimal places the number or elements in the array will be rounded to. The default value is 0. This parameter is optional and will accept negative integers as well as positive ones.
+- `out` is an optional output array which the result will feed into.
+
+> **Note:** Numpy is commonly imported as `import numpy as np`. This allows us to refer to `numpy` as `np`.
```

Comment:
```suggestion
```

Reply:
This note is not needed, it can be removed since it is covered in the concept entry of numpy